nerfstudio/models/splatfacto.py

```diff
@@ -747,10 +747,14 @@ def get_outputs(self, camera: Cameras) -> Dict[str, Union[torch.Tensor, List]]:
             tile_bounds,
         )  # type: ignore
 
+        # rescale the camera back to original dimensions before returning
+        camera.rescale_output_resolution(camera_downscale)
```

Comment:
You need to remove the line 773-774

Reply:
Ah yes, thanks for catching -- just fixed this